apps/settings/js/panels/display/slider_handler.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Update setting value based on slider position.
+ * Base on sound panel's implementation.
+ *
+ * @module SliderHandler
+ */
+define(function(require) {
+  'use strict';
+
+  const INTERVAL = 300;
+
+  var SliderHandler = function() {
+    this._element = null;
+    this._key = '';
+    this._isFirstInput = false;
+    this._intervalID = null;
+  };
+
+  SliderHandler.prototype = {
```

Comment:
Please add a README in both display & sound panel to notice maintainer that we need sync these 2 SliderHandler